docs/docs/guides/how-to-parameterize-cassandra.md

```diff
@@ -0,0 +1,396 @@
+---
+title: How to set up an n-node Cassandra environment
+sidebar_label: Setting up an n-node Cassandra cluster
+slug: /how-to-parameterize-cassandra
+toc_max_heading_level: 2
+sidebar_position: 5
+---
+
+Introduction
+------------
+In this guide, we will set up a 3-node Cassandra cluster in Docker and parameterize the environment definition so it can easily be modified for use in different tests that require an _n_-node Cassandra cluster. Then we will show you how to run remotely hosted packages authored by others, and go through how to package and publish our work to Github for others to use as well.
+
+Specifically, we're going to configure our test environments with a way that allows us to both:
+1. Parameterize the environment so a user can trivially specify how many nodes they’d like for their system to have, and 
+2. Make their environment definition composable so that these environments can be easily included in tests with other services for different scenarios & use cases.
+
+**Without Kurtosis**
+
+One way to accomplish the above would be to write shell scripts over docker, or over binaries running on bare metal. In this case, we’d have to build these capabilities into our scripts and handle cross-system issues (laptop vs CI) ourselves.
+
+**With Kurtosis**
+
+However, in this guide, we’re going to use a free tool that already has these capabilities built in - Kurtosis. Kurtosis is a composable build system for writing reproducible test environments, and can run on your own laptop or in your favorite CI provider.
+
+:::info
+For a holistic view of Kurtosis’ capabilities, visit our [Quickstart][quickstart]!
```

Comment:
I'd suggest just removing this info box, they have it in the sidebar

Reply:
Done